docs/tutorials/_index.md

```diff
@@ -3,4 +3,62 @@ title: "Tutorials"
 linkTitle: "Tutorials"
 weight: 65
 type: docs
+no_list: true
 ---
+
+{{< alert title="Tip" color="tip" >}}
+While tutorials may list specific hardware, the hardware only affects the configuration of a project.
+Once you have configured your components, you can use the same code of the tutorials with any hardare.
+
+For this reason, we list tutorials based on features you may be looking for.
+{{< /alert >}}
+
+## Your first robots
+
+{{< cards >}}
+    {{< card link="/tutorials/get-started/try-viam-sdk" size="small">}}
+    {{< card link="/tutorials/get-started/make-an-led-blink-with-the-viam-app/" size="small" >}}
+    {{< card link="/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk/" size="small" >}}
+{{< /cards >}}
+
+## Configuration and Testing
+
+{{< cards >}}
+    {{< card link="/tutorials/configure/build-a-mock-robot/" size="small" >}}
```

Comment:
The GIF for this one has a weird shadow on the left side that comes and goes (and I think has something to do with an adjacent window casting a graphic shadow on the creator's screen)

Reply:
I'm aware - I don't want to re-record this. When we refactor this tutorial we can address that